wfe2/verify.go

```diff
@@ -315,6 +351,15 @@ func (wfe *WebFrontEndImpl) extractJWK(jws *jose.JSONWebSignature) (*jose.JSONWe
 	return key, nil
 }
 
+// jwsKeyIDToAccountID returns the KeyID string from a JWS. It should be called
+// only using JWS from `parseJWS` to ensure the correct number of Signatures are
+// present.
+func (wfe *WebFrontEndImpl) jwsKeyIDToAccountID(jws *jose.JSONWebSignature) string {
+	header := jws.Signatures[0].Header
```

Comment:
I think originally I was also stripping off the reg prefix here and it felt more worthwhile. I think it's probably safe to cut now that it's just pulling out the KeyID as-is.